src/main/java/io/lighty/yang/validator/exceptions/NotFoundException.java

```diff
@@ -1,7 +1,14 @@
 package io.lighty.yang.validator.exceptions;
 
+/**
+ * This exception is thrown when value of Optional variable is not present.
+ * This exception is for handle 3 types of cases when module is not found,
+ * revision of module is not found or when some node is missing.
+ * Takes two parameters. First is customMessage. It should describe what is not found(module, revision or node)
+ * Second parameter is name of this what is not found.
```

Comment:
Don't be so specific. For better maintainability, it will be better to describe it more abstract - something like, 
"indicating that expected element was not found by lyv, for example when expected `revision` or  `DataSchemaNode` was not found"